src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,92 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+
+public class HomePage extends BasePage{
+    public HomePage(WebDriver givenDriver) {
+        super(givenDriver);
+    }
+    //Verify that the song is playing and the sound bar is moving
+
+    @FindBy(css = "[data-testid='sound-bar-play']")
+    WebElement soundBars;
+    @FindBy (css = "[data-testid='play-btn']")
+    WebElement playSong;
+    @FindBy (css = ".side.player-controls")
+    WebElement playPanel;
+    @FindBy (css = ".ok")
+    WebElement clickOk;
+    @FindBy (css = "button[title='Delete this playlist']")
+    WebElement deletePlaylist;
+    @FindBy (css = "[class='playlist playlist']")
+    WebElement selectPlaylist;
+    @FindBy (css = "[name='name']")
+    WebElement playListInputField;
+    @FindBy (css = "[class='playlist playlist']")
+    WebElement doubleClick;
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```
   @FindBy(css = "img.avatar")
    private WebElement avatar
```

![POM_vs_Factory_1](https://github.com/Test-Pro-Bootcamp/A49-QA-Automation-Framework/assets/104761482/81d9c549-f3db-438c-a12f-49272d2117da)

![POM_vs_Factory_2](https://github.com/Test-Pro-Bootcamp/A49-QA-Automation-Framework/assets/104761482/5a558ab3-8e7a-45de-8719-0bcb3aa38806)